pymc/variational/opvi.py

```diff
@@ -1693,18 +1465,6 @@ def ndim(self):
     def ddim(self):
         return sum(self.collect("ddim"))
 
-    @property
-    def has_local(self):
-        return any(self.collect("local"))
-
-    @property
-    def has_global(self):
-        return any(not c for c in self.collect("local"))
-
-    @property
-    def has_batched(self):
-        return any(not c for c in self.collect("batched"))
-
     @node_property
     def symbolic_random(self):
         return at.concatenate(self.collect("symbolic_random2d"), axis=-1)
```

Comment:
symbolic_random2d is no more needed, no 3d tensor randoms to concatenate
```suggestion
        return at.concatenate(self.collect("symbolic_random"), axis=-1)
```

Reply:
done